### ISSUE TITLE
SC2142 false positive with inline functions inside aliases

### ISSUE THREAD

**Issue by ssbarnea** [koalaman/shellcheck#3159]
We should not need to disable SC2142 in order to make use of inline functions, example:

```shell
# shellcheck disable=SC2142
alias mypy='f() { command mypy "${@:-.}"; }; f'
```

Code above adds `.` argument to mypy if is called without any arguments.
